application/src/main/java/org/togetherjava/tjbot/commands/help/BookmarksCommand.java

```diff
@@ -0,0 +1,161 @@
+package org.togetherjava.tjbot.commands.help;
+
+import net.dv8tion.jda.api.entities.MessageEmbed;
+import net.dv8tion.jda.api.entities.channel.unions.MessageChannelUnion;
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent;
+import net.dv8tion.jda.api.interactions.commands.OptionMapping;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.togetherjava.tjbot.commands.CommandVisibility;
+import org.togetherjava.tjbot.commands.SlashCommandAdapter;
+
+import java.util.Objects;
+
+/**
+ * The bookmarks command is used for managing and viewing bookmarks. A bookmark is a link to a help
+ * thread that can have a note so you can easily remember why you bookmarked a help thread. Writing
+ * to the database and showing the list/remove messages is not done by this class, that is handled
+ * by the {@link BookmarksSystem}. This class only checks if you are able to add a bookmark in the
+ * current channel and tells the {@link BookmarksSystem} to do the rest.
```

Comment:
give an overview of all the available commands. see for example `TagCommand` for an example of how to write that nicely in Javadoc

Reply:
I changed it the way AskCommand does